pretrain/tinyllama.py

```diff
@@ -222,7 +222,7 @@ def train(fabric, state, train_dataloader, val_dataloader, resume):
             }
 
             fabric.print(
-                f"iter {metrics['iter']} step {metrics['step']}: loss {metrics['loss']:.4f}, iter time:"
+                f"iter {metrics['iter']} | epoch {train_iterator.epoch} | step {metrics['step']}: loss {metrics['loss']:.4f}, iter time:"
```

Comment:
Since epochs are very long, it would be less noisy to only print the epoch when it changes instead of on every loss print.

Feel free to ignore if you disagre.

Reply:
On the other hand, if the epochs are very long, you'll need to scroll the log very long to find the epoch print.

Alternatively, I could remove it from the print and log it instead if you prefer that more.

Reply:
I like that suggestion. Usually the terminal width is very limited